utils/config/daos.yml

```diff
@@ -1,42 +1,44 @@
-# DAOS agent configuration file.
+# DAOS client configuration file.
 #
 # Location of this configuration file is determined by first checking for the
-# path specified through the -f option of the daos_agent command line.
-# Otherwise, /etc/daos.conf is used.
+# path specified through the -o option of the daos_agent command line.
+# Otherwise, /etc/daos.yml is used.
 #
 # Section describing the client configuration
 #
 # Although not supported for now, one might want to connect to multiple
 # DAOS installations from the same node in the future.
 #
 # Specify the associated DAOS systems.
-# Name must match name specifed in the daos_server.conf file on the server.
-#
-systems:
--
-  name: daos
-#
-#  # Access points
-#  # To operate, DAOS will need a quorum of access point nodes to be available.
-#  # Hosts can be specified with or without port, default port below.
-#  #
-#  # access_pts: ['hostname1:10001','hostname2:10001','hostname3:10001']
-#  # access_pts: [hostname1,hostname2,hostname3]
-#
-#  # Force default port number to connect to on daos_server, this will also
-#  # be used when connecting to access points if no port is specified.
-#  #
-#  # default: 10000
-#  # port: 10001
-#
-#  # Path to CA certificate
-#  # Not required if the DAOS server is configured in an insecured way
-#  #
-#  # ca_cert: ./.daos/ca.crt
-#
-#  # Path to client certificate and key file.
-#  # Discarded if no CA certificate is passed.
-#  #
-#  # default: ./.daos/daos.{crt,key}
-#  # cert: ./.daos/daos.crt
-#  # key: ./.daos/daos.key
+# Name must match name specifed in the daos_server.yml file on the server.
+#
+# default: daos
+name: daos
+
+# Management server access points
+# default: hostname of this node at port 10001 for local testing
+#access_points: ['hostname1:10001','hostname2:10001','hostname3:10001']
+#access_points: [hostname1,hostname2,hostname3]
+
+# Force different port number to connect to access points.
+# default: 10000
+#port: 10001
+
+# Hostlist
+# comma separated list of addresses <ipv4addr/hostname:port>
+# default: ['localhost:10001']
+#hostlist: ['localhost:10001']
+
+
+# Path to client certificate and key file.
+# Discarded if no CA certificate is passed.
+# default: ./.daos/daos.{crt,key}
+#cert: ./.daos/daos.crt
+#key: ./.daos/daos.key
+
+# Use the given directory for creating unix domain sockets
+# default: /var/run/daos_agent
+runtime_dir: /var/run/daos_agent
+
+# default: /tmp/daos_agent.log
```

Comment:
Probably want a comment of explanation here too

Reply:
Will do.